docs/source/usage/Ellipsoid.rst

```diff
@@ -0,0 +1,117 @@
+.. _transformation:
+
+================================================================================
+Ellipsoid  transformation
+================================================================================
+A ellipsoid is the mathematically defined surface which approximates the geiod, surface of the earth's gravity field, which is approximately the same as mean sea level.
+
+
+.. figure:: ./images/General_ellpsoid.png
+   :width: 500 px
+   :align: center
+   :alt:   Global and local fitting of the ellipsoid
+
+   proj-string: ``+proj=merc``
+
+
+
+A complete ellipsoid definition comprises a size (primary) and a shape (secondary)
+parameter.
+
+Ellipsoid Size parameters 
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+.. _ellipsoid_parameters : PROJ/docs/source/operations/options/
+
+	============   ==============================================================
+	Parameter      Description
+	============   ==============================================================
+	+R             Radius of a spherical planet
+	+a             Semi-major axis of an ellipsoidal planet
+	============   ==============================================================
```

Comment:
```suggestion
.. option:: +R=<value>

    Radius of a spherical planet

.. option:: +a=<value>

    Semi-major axis of an ellipsoidal planet
```

Also, I would probably avoid using the term "planet" as someone might want to model a celestial bodies that aren't planets. "Object" is probably a better word but someone may have a better suggestion than that.